src/cleSetNonzeroPixelsToPixelindexKernel.cpp

```diff
@@ -23,6 +23,12 @@ void SetNonzeroPixelsToPixelindexKernel::SetOutput(Object& x)
     this->AddObject(&x, "dst");
 }
 
+void SetNonzeroPixelsToPixelindexKernel::SetOffset(float x)
```

Comment:
the offset in the kernels is expecting an int, not a float, this can explain the failing

Reply:
YES! 🙌 Thanks for spotting this :-)